lib/index.js

```diff
@@ -9,6 +9,20 @@ if (typeof Promise === "undefined") {
   require("es6-promise").polyfill();
 }
 
+if (typeof Buffer === "undefined") {
+  const g = global || window;
+  g.Buffer = require("buffer/").Buffer;
+}
+
+if (typeof process === "undefined") {
+  const g = global || window;
+  g.process = require("process/");
```

Comment:
same as line 14 above.

Reply:
I tested this a bit more, and you are right. The trailing slash can be removed. I'll make the change.